docs/source/datasetinsights.commands.rst

```diff
@@ -0,0 +1,34 @@
+datasetinsights.commands
```

Comment:
Don't we do `make clean` before we merge to master?

Reply:
make clean is not required as html files are ignored. RST files has to be checked in for readthedocs to build automatically.